README.md

```diff
@@ -100,3 +100,11 @@ to create one in the Basis Theory portal.
 
 A full example Android app is included within the [example](example) module within this repo.
 
+## Security Notice
+
+This SDK includes various measures to protect sensitive data entered into secure elements and to
+prevent unauthorized access. However, it is important to note that due to the inherent flexibility
+of the Android platform, it is not possible to fully prevent all forms of data access by a developer
```

Comment:
is it really just because of the flexibility of Android or also because a malicious developer can literally send the sensitive data to a mocked element he made or anywhere else. In other words, a malicious dev can really do whatever they want and lie to the company saying they are using our SDK

Reply:
I definitely agree with this point: that there are limitless ways a malicious developer could steal the data. I thought, though, we are trying to point to some platform limitations on Android that don't exist on iOS, for example. I'm totally comfortable to broaden this language to simply say: our tools make it easy to collect this data in a touchless way, but if the developers want to touch the data, we can't prevent them from doing so.